added posts for ics (#5)
hyphacoop/remote: _posts/2020-03-20-crash-course-on-working-remote.md

---
layout: post

> title: "crash course on working remote"

Reviewer: ```suggestion
title: "Crash course on working remote"
```

> slug: meet-and-greet

Reviewer: ```suggestion
slug: working-remote
```

> text: 'Anyone can join us on Thursdays @ 11:30-12:30 ET for the next three weeks to discuss collective solutions to challenges that arise when working remotely'

Reviewer: ```suggestion
text: 'Anyone can join us at by videoconferencing to discuss collective solutions to challenges that arise when working remotely. See https://covid19.hypha.coop for details.'
```

location: Jitsi

> locationLink: https://jitsi.hypha.coop/COVID19TorontoRemote

Reviewer: ```suggestion
location: https://jitsi.hypha.coop/COVID19TorontoRemote
```

date: 2020-03-20
startTime: '11:30'
endTime: '12:30'
---

Anyone can join us at by videoconferencing to discuss collective solutions to challenges that arise when working remotely. See
https://covid19.hypha.coop

> for details.

Reviewer: ```suggestion
```
